src/toolbar/types.ts

```diff
@@ -111,3 +121,12 @@ export type ToolbarReactNodeFnData<E> = {
     width: number;
     content: (e: E) => React.ReactNode;
 };
+
+export type WToolbarData = ToolbarData<ActionStorage>;
+export type WToolbarItemData = ToolbarItemData<ActionStorage>;
+export type WToolbarSingleItemData = ToolbarSingleItemData<ActionStorage>;
+export type WToolbarGroupData = ToolbarGroupData<ActionStorage>;
+export type WToolbarGroupItemData = ToolbarGroupItemData<ActionStorage>;
+export type WToolbarListButtonData = ToolbarListButtonData<ActionStorage>;
+export type WToolbarListItemData = ToolbarListItemData<ActionStorage>;
+export type WToolbarListButtonItemData = ToolbarListButtonItemData<ActionStorage>;
```

Comment:
this is not part of `src/toolbar` module, so this should be in `src/bundle/toolbar/types.ts`

Reply:
I'm sorry I can't do that. I creates circular dependency `  03) src/toolbar/index.ts -> src/toolbar/FlexToolbar.tsx -> src/bundle/toolbar/types.ts ` 

If it's ok i would leave it here for now.

Reply:
But it's just types o_O
They don't creates circular deps in js code